widgets/src/main/java/org/pentaho/gwt/widgets/client/listbox/CustomListBox.java

```diff
@@ -526,7 +526,7 @@ private void updateDropDown() {
     } else {
       editableTextBox.setTabIndex( -1 );
       fPanel.removeStyleName( "focus-container" );
-      fPanel.setTabIndex( 0 );
+      fPanel.setTabIndex( ( this.enabled ) ? 0 : -1 );
```

Comment:
No need for the extra parenthesis.
Also, please fix the same issue for the editable case, 6 lines above, on line 527.